suggestions/cloud_services.json

```diff
@@ -1 +1,30 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "com.owncloud.android",
+        "label": "ownCloud",
+        "source": "fg",
+        "reason": "Add an ownCloud server now and get... Your Cloud, Your Data, Your Way!\nhttps://owncloud.org/",
+        "repo": "https://github.com/owncloud/android"
+    },
+    {
+        "id": "com.infomaniak.drive",
+        "label": "Infomaniak kDrive",
+        "source": "fg",
+        "reason": "The swiss cloud service that doesn’t analyse or resell stored data.\nhttps://www.infomaniak.com/\nContains crash reports telemetry bits",
+        "repo": "https://github.com/Infomaniak/android-kDrive"
+    },
```

Comment:
Has opt-in trackers, promotes non-free network services. The other candidates are simply better than this.